data/hai/hai outfits.txt

```diff
@@ -59,6 +59,32 @@ outfit "Pulse Turret"
 		"hull damage" 34
 	description "Pulse Cannons are the primary offensive weapon used by Hai ships. This turret mounts two pulse cannons side by side and allows them to track even the fastest-moving of targets."
 
+outfit "Tripulse Shredder"
+	category "Guns"
+	cost 520000
+	thumbnail "outfit/tripulse shredder"
+	"mass" 42
+	"outfit space" -42
+	"weapon capacity" -42
+	"gun ports" -1
+	weapon
+		sprite "projectile/pulse"
+		sound "pulse"
+		"hit effect" "pulse impact"
+		"inaccuracy" 5
+		"velocity" 15
+		"lifetime" 30
+		"reload" 6
+		"burst count" 36
+		"burst reload" 4
+		"cluster"
+		"firing energy" 25
+		"firing heat" 60
+		"shield damage" 45
+		"hull damage" 34
+	description "An early stage in the development of better and faster weaponry to rival that of the Wanderers, the Tripulse Shredder is essentially just three Pulse Cannons in one single, larger housing. Integrating advances in cooling capabilities has reduced the heat production per shot significantly at the cost of drawing just a touch more power. The quicker firing speed, however, still generates much more total heat in the same time. Additionally, the single shared housing results in a less stable firing pattern, limiting it to bursts with short cooldown gaps in between."
+	description "	Initially designed with the Sea Scorpion in mind, the Tripulse Shredder's unique properties compared to the standard Pulse Cannon make it ideal for hit and run tactics, and the Unfettered pilots most skilled with this weapon are able to time its needed rest time to sync with the end of their initial attack, allowing them to harass freighters or even cripple their smaller escort ships with its high damage before either fleeing or coming about for a second pass. Furthermore, when this technique is utilized properly, the weapon allows for small, agile ships to damage and even defeat heavier ships by making use of their high mobility to compensate for its low range."
```

Comment:
These weapons are going to be in the campaign, and will be not be unlocked at any time, you can only steal them (and are encouraged to do so)
I like long descriptions because nobody has to read them but they add depth to the game